lib/plugins/pump.js

```diff
@@ -0,0 +1,265 @@
+'use strict';
+
+var _ = require('lodash');
+var moment = require('moment');
+var levels = require('../levels');
+var timeago = require('./timeago')();
+
+var ALL_STATUS_FIELDS = ['reservoir', 'battery', 'clock', 'status'];
+
+function init ( ) {
+
+  var pump = {
+    name: 'pump'
+    , label: 'Pump'
+    , pluginType: 'pill-status'
+  };
+
+  pump.getPrefs = function getPrefs (sbx) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (11)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1808382488/issues/source?bid=2452379&fileBranchId=3051223#l18)